Bio/PDB/internal_coords.py

```diff
@@ -1933,23 +1932,23 @@ def pick_angle(
         The following are equivalent (except for sidechains with non-carbon
         atoms for chi2):
 
-                ric = r.internal_coord
-                print(
-                    r,
-                    ric.get_angle("psi"),
-                    ric.get_angle("phi"),
-                    ric.get_angle("omg"),
-                    ric.get_angle("tau"),
-                    ric.get_angle("chi2"),
-                )
-                print(
-                    r,
-                    ric.get_angle("N:CA:C:1N"),
-                    ric.get_angle("-1C:N:CA:C"),
-                    ric.get_angle("-1CA:-1C:N:CA"),
-                    ric.get_angle("N:CA:C"),
-                    ric.get_angle("CA:CB:CG:CD"),
-                )
```

Comment:
I think this indentation was wrongly removed - RST would have used it to define a block, a literal block if the preceding paragraph ended with a double colon. Certainly the flake8 complaints fall here.